documentation/docs/integrations/webhooks.md

```diff
@@ -4,4 +4,39 @@ title: Webhooks Integration
 sidebar_label: Webhooks
 ---
 
-More information on how to integrate with webhooks is coming soon. If you’re interested in this option, please see or open a [GitHub Issue](https://github.com/monosidev/monosi/issues).
+Monosi integrates directly with Slack to be alert you in real-time of anomalies. It does so through the Incoming Webhooks feature of Slack.
```

Comment:
This page is not the same as the slack incoming webhook

Reply:
fixed